packages/actions/index.ts

```diff
@@ -1026,3 +1026,9 @@ const settleBilateral = async (user, bilateralSettlements, dispatch, getState) =
     }
   })
 }
+
+export const showPayPalSettlementError = (nickname: string) => {
+  return async (dispatch, _getState) => {
+    dispatch(displayError(settlementManagement.bilateral.error.generic(nickname)))
```

Comment:
I'm not getting a Toast error popup or any other message to the user.

Reply:
I'll work on this